xbmc/addons/kodi-dev-kit/include/kodi/c-api/addon-instance/inputstream/stream_constants.h

```diff
@@ -0,0 +1,50 @@
+/*
+ *  Copyright (C) 2017-2020 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
```

Comment:
I don't believe `#pragma once` is appropriate C. Header guards are the idiomatic way to prevent double inclusion of C headers

Reply:
It's good if I'm wrong, because I much prefer `#pragma once`

Reply:
There I forget on most headers to add, e.g.:
```c
#ifndef C_API_ADDONINSTANCE_INPUTSTREAM_STREAMCONSTANTS_H
#define C_API_ADDONINSTANCE_INPUTSTREAM_STREAMCONSTANTS_H

#endif /* !C_API_ADDONINSTANCE_INPUTSTREAM_STREAMCONSTANTS_H */
```